src/components/Card/components/Section/Section.tsx

```diff
@@ -6,28 +6,26 @@ import Subheading from '../../../Subheading';
 import * as styles from '../../Card.scss';
 
 export interface Props {
-  title?: string;
+  title?: React.ReactNode;
   children?: React.ReactNode;
   subdued?: boolean;
   fullWidth?: boolean;
 }
 
 export default function Section({children, title, subdued, fullWidth}: Props) {
-  const headerContent = title ? (
-    <div className={styles.SectionHeader}>
-      <Subheading>{title}</Subheading>
-    </div>
```

Comment:
What's the reason behind deleting the `<div className={styles.SectionHeader}>` here? Seems like something we would want kept in

Reply:
Correct, looks like this was the cause of the percy failures